### PR TITLE
Broll/ci

### PR DESCRIPTION
**Pull request by mrbroll**
This adds a Dockerfile and CI via travis. The release portion of the travis setup has not been tested yet, since there won't be any tagged commits until everything gets merged to master.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: go
+go:
+- 1.8
```
Comment (BlueDragonX) [inline .travis.yml]: Use 1.9

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+services:
+- docker
+before_install:
+- go get github.com/mitchellh/gox
+- go get github.com/golang/lint/golint
+install: true
+script:
+- go fmt . ./prometheus/...
+- go vet . ./prometheus/...
+- golint . ./prometheus/...
```
Comment (BlueDragonX) [inline .travis.yml]: You can use `go list ./...` to build a list of packages for these commands. For instance:
```
go vet $(go list ./...)
golint | xargs go list -f '{{.Dir}}' ./...
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- 1.8
+sudo: required
+services:
+- docker
+before_install:
+- go get github.com/mitchellh/gox
+- go get github.com/golang/lint/golint
+install: true
+script:
+- go fmt . ./prometheus/...
```
Comment (BlueDragonX) [inline .travis.yml]: This only formats the code. It always returns a 0 exit code so will not fail tests if formatting is needed.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- go vet . ./prometheus/...
+- golint . ./prometheus/...
+- go test -v ./prometheus/...
```
Comment (BlueDragonX) [inline .travis.yml]: The standard Go tooling takes `./...` as an argument. So this can be simplified:
```
go test -v ./...
```

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,7 @@
+FROM debian:jessie
+
+RUN mkdir -p /opt/bin
+
+ADD kafka-connect-exporter /opt/bin
```
Comment (BlueDragonX) [inline Dockerfile]: Just put it in `/usr/bin/` and you can avoid the `mkdir` call and use simply `ENTRYPOINT ["kafka-connect-exporter"]`.

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+deploy:
+  provider: releases
+  api_key:
+    secure: Zy5UE+7Z6DhIr0/X8Nrola+ZpoEjCb8xV9+kBNCjb+2bBMK9Wiu6R2AeLDWS/meIKeJbJ/ffUAVBuRmVwB0lUQhi9DnPtaF91BSL3ytfWb/vHZsIumc7Upbpt59ysczgT3AtfXv2GSkYAf9r1rejuoiSp60p/MiF97VHaNOhEWnSWhw1lVKW3lDAonE1BGsaqrVavZymQ7UjtJQUeB8x3+kMgcJPxSY5dC6TImvn76yMZp/Vioovl2+qmQNu+g8paXct1KtJ9aAF5o44hi56uJ+L4xPTzrxsP0LZhLn6C80VSlQCpd4FGdmzgdxMPa/Gn3PymvO68vPW+9Qax+Unri8wWOtk9nBE65/d67nJWlxgVEagjeB8vHJGhELVsKUM4nWxphAgXjH3R5U3riTiTGSF4QXsY5ZoiouZ99D9/FaglOvw3satqGDB0rmpcUseECPZWqKw6RuHvbOcvclAa3rcR7epYeyilgOGiREkE92VCBwvBHusxryIMkOEe3G1MWzJxra7/7GrRRc3qAgJlY19Eq5/tXpZMota4o3p36nvlp0wVKyOhFnjud/Ta0h/d0/CwOvepnTSyY3TqsaWyyPBQV4SSfHT8nc1dCVC2uenHyYTqjyV3vTtbvRf84umhuOe7ycTJ3JoKM+If56RAlaljFE0//lhnd6dWcic+JU=
+  skip_cleanup: true
+  file_glob: true
+  file: dist/*
+  on:
+    tags: true
+    repo: zenreach/kafka-connect-exporter
```
Comment (BlueDragonX) [inline .travis.yml]: Make sure you've given the `Releases` group write permission to the repo.

File: .travis.yml (hunk 3 above)
Comment (mrbroll) [reply .travis.yml]: Ugh, yeah I forgot it's not the same as `gofmt`.

File: .travis.yml (hunk 6 above)
Comment (mrbroll) [reply .travis.yml]: done

**Pull request merged**